contrib/gonic/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "gonic"
+pkgver = "0.16.4"
+pkgrel = 0
+build_style = "go"
+make_build_args = ["cmd/gonic/gonic.go"]
+hostmakedepends = ["go", "pkgconf"]
+makedepends = ["sqlite-devel", "taglib-devel"]
+go_build_tags = ["libsqlite3"]
+pkgdesc = "Music streaming server / subsonic server API implementation"
+maintainer = "samg <samg@fastmail.com>"
+license = "GPL-3.0-only"
+url = "https://github.com/sentriz/gonic"
+source = f"https://github.com/sentriz/gonic/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "ba690a861a075bdf3f1f7e8131e45a5ca430ec90902a97faf955ec9f36799461"
+# tests require network access, transcoding test requires ffmpeg
+options = ["!check", "!cross"]
```

Comment:
```suggestion
# taglib gomod can't cross
options = ["!cross"]
```

Reply:
(the default go test ./... seems to work fine)

Reply:
I'm getting a test failure in when building in the sandbox but not outside of it; still trying to sort it out

Reply:
i'm not sure how you're testing that but just running a regular `pkg` with this works fine for me, just push it and see what CI says

Reply:
hm ok seems to fail, why did it work for me...

Reply:
and it definitely runs the same failing test for me locally

```
ok  	go.senan.xyz/gonic/server/ctrlsubsonic	0.101s
```

nothing in the files seem to e.g. do something different based on CI in env or whatnot so i'm very confused..

Reply:
aand now it works

so i guess that's actually a flake that only sometimes happens, oh well